web_src/js/index.js

```diff
@@ -2240,6 +2335,17 @@ function initTemplateSearch() {
 }
 
 $(document).ready(async () => {
+  // Show issue or pr in board sidebar
+  $('.board-column').on('click', '.draggable-cards [data-url]', (e) => {
+    e.preventDefault();
+    $('#current-card-details').html('');
+    $('#current-card-details-input').html('');
+    const data = $(e.currentTarget).data();
+    $('#current-card-details').css('visibility', 'visible');
+    $('#current-card-details').show();
+    reloadIssuesActions(data);
+  });
```

Comment:
This can be moved to `projects.js` after moving `reloadIssuesActions` to its own file too and exporting it there.

Reply:
I tried this but it turned out it depends on 60 to 70% of index.js, so I end up moving one big file to another big file. It makes sense to refactor index.js but that's for another PR IMO.